urbanairship-react-native/js/UrbanAirship.ts

```diff
@@ -734,21 +745,31 @@ export class UrbanAirship {
   /**
    * Gets the subscription lists.
    *
-   * @param types The types of subscription lists. Defaults to an [`channel`].
+   * @param types The types of subscription lists. Values: `channel`, `contact` (default: [`channel`]).
    * @return A promise with the result.
    */
-   static getSubscriptionLists(types?: [SubscriptionListType]): Promise<SubscriptionLists> {
+   static getSubscriptionLists(types?: [...SubscriptionListType[]]): Promise<SubscriptionLists> {
```

Comment:
We need to make sure the edit, get, and get preference all use the same type. I think its lower case everywhere